ale_linters/json/jsonlint.vim

```diff
@@ -0,0 +1,44 @@
+" Author: KabbAmine <amine.kabb@gmail.com>
+
+if exists('g:loaded_ale_linters_json_jsonlint')
+    finish
+endif
+
+let g:loaded_ale_linters_json_jsonlint = 1
+
+function! ale_linters#json#jsonlint#Handle(buffer, lines)
+    " Matches patterns like the following:
+    " line 2, col 15, found: 'STRING' - expected: 'EOF', '}', ',', ']'.
+
+    let pattern = 'line \(\d\+\), col \(\d*\), \(.\+\)'
```

Comment:
I recommend writing `^` and `$` most of the time to write a regex for the entire line. This should work, though.